File: CodingExamples/CodingExamples/_2-The_Sum_Of_The_Digits_Of_The_Number/Java/SumDigits.java
import java.util.Scanner;
class SumDigits {
    public static void main(String[] args) {
        Scanner input=new Scanner(System.in);
        int num,result=0,i=10;
    do{
    result=0;
    System.out.print("Please enter a number : ");
    num=input.nextInt();
    System.out.println();
    if(num>0){
    while(num>0){
        result+=num%i;
        num/=10;
    }
    System.out.println("Result = "+ result+"\n");
    }else{
        System.out.println("Invalid Process!Please enter a positive number!");
    }
    }while(result>10);
    }
}

File: CodingExamples/CodingExamples/_2-The_Sum_Of_The_Digits_Of_The_Number/C/sumdigits.c
#include <stdio.h>
int main() {
    int num,result=0,i=10;
    do{
    result=0;
    printf("Please enter a number : ");
    scanf("%d",&num);
    if(num>0){
    while(num>0){
        result+=num%i;
        num/=10;
    }
    printf("Result = %d\n",result);
    }else{
        printf("Invalid Process!Please enter a positive number!\n");
    }
    }while(result>10);
    return 0;
}